tests/gen/memcache/gen_all.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
```

Comment:
No, don run this every time. This script is about generating the different scenarios:

Reasons not to run every single time:
- you need a memcache instance to create pcaps => commit generated pcaps
- You need privileges to sniff traffic via tcpdump
- Also running every single time would make tests unstable as UDP can drop packets (even between local VMs as I've found)

Will add a README explaining how to use it